src/buildercore/cfngen.py

```diff
@@ -171,6 +145,43 @@ def _parameterize(string):
     return context
 
 
+def build_context_rds(context, existing_context):
```

Comment:
seems a good way to express only RDS has a dependency on `existing_context`

Reply:
that's what I thought